src/Activity.js

```diff
@@ -67,51 +67,51 @@ class Activity {
-    return winner;
-  }
-  getStreak(userRepo, id, relevantData) {
-    let data = this.activityData;
-    let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
-    let streaks = sortedUserArray.filter(function(element, index) {
-      if (index >= 2) {
-        return (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData])
-      }
-    });
-    return streaks.map(function(streak) {
-      return streak.date;
-    })
-  }
-  getWinnerId(user, date, userRepo) {
-    let rankedList = this.getFriendsAverageStepsForWeek(user, date, userRepo);
-    let keysList = rankedList.map(listItem => Object.keys(listItem));
-    return parseInt(keysList[0].join(''))
-  }
+  // getFriendsActivity(user, userRepo) {
+  //   let data = this.activityData;
+  //   let userDatalist = user.friends.map(function(friend) {
+  //     return userRepo.getDataFromUserID(friend, data)
+  //   });
+  //   return userDatalist.reduce(function(arraySoFar, listItem) {
+  //     return arraySoFar.concat(listItem);
+  //   }, []);
+  // }
+  // getFriendsAverageStepsForWeek(user, date, userRepo) {
+  //   let friendsActivity = this.getFriendsActivity(user, userRepo);
+  //   let timeline = userRepo.chooseWeekDataForAllUsers(friendsActivity, date);
+  //   return userRepo.combineRankedUserIDsAndAveragedData(friendsActivity, date, 'numSteps', timeline)
+  // }
+  // showChallengeListAndWinner(user, date, userRepo) {
+  //   let rankedList = this.getFriendsAverageStepsForWeek(user, date, userRepo);
+
+  //   return rankedList.map(function(listItem) {
+  //     let userID = Object.keys(listItem)[0];
+  //     let userName = userRepo.getDataFromID(parseInt(userID)).name;
+  //     return `${userName}: ${listItem[userID]}`
+  //   })
+  // }
+  // showcaseWinner(user, date, userRepo) {
+  //   let namedList = this.showChallengeListAndWinner(user, date, userRepo);
+  //   let winner = this.showChallengeListAndWinner(user, date, userRepo).shift();
+  //   return winner;
+  // }
+  // getStreak(userRepo, id, relevantData) {
+  //   let data = this.activityData;
+  //   let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
+  //   let streaks = sortedUserArray.filter(function(element, index) {
+  //     if (index >= 2) {
+  //       return (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData])
+  //     }
+  //   });
+  //   return streaks.map(function(streak) {
+  //     return streak.date;
+  //   })
+  // }
+  // getWinnerId(user, date, userRepo) {
+  //   let rankedList = this.getFriendsAverageStepsForWeek(user, date, userRepo);
+  //   let keysList = rankedList.map(listItem => Object.keys(listItem));
+  //   return parseInt(keysList[0].join(''))
+  // }
 }
```

Comment:
Nice work with this significant reduction in code. By getting rid of these unnecessary methods its easier to understand the functionality of each class. Let's keep these commented out for the time being in case we need to refer to them later - we can delete before project submission.